content/post/practical-tb.Rmd

```diff
@@ -446,13 +446,13 @@ out0  <- get_intervention(xstart, params, NA, times, NA, NA, TB.Basic,
                           "Initial", NA) 
 
 # plot
-dot <- data.frame(Data = "Bangladesh", Years = 2017, incidence = Inc.country)
```

Comment:
Hi @juanvesga,

This produces an error. Was this supposed to change to `country.name`?